### ISSUE TITLE
executing failed to find the calling node on 3.10

### ISSUE THREAD

**Issue by samuelcolvin** [alexmojaki/executing#40]
As per [this](https://github.com/samuelcolvin/python-devtools/actions/runs/2738231698) run of devtools CI, [this](https://github.com/samuelcolvin/python-devtools/blob/54b2b2b672bb76dfb14f875b4b41957162bb257f/tests/test_main.py#L180-L188) test is failing.

The culprit is [this](https://github.com/samuelcolvin/python-devtools/blob/54b2b2b672bb76dfb14f875b4b41957162bb257f/devtools/debug.py#L170-L171), code:

```py
        source = executing.Source.for_frame(call_frame)
        if not source.text:
            warning = 'no code context for debug call, code inspection impossible'
            arguments = list(self._args_inspection_failed(args, kwargs))
        else:
            ex = source.executing(call_frame)
            function = ex.code_qualname()
            if not ex.node:  <-- ex.node is falsey causing the error
                warning = "executing failed to find the calling node"
```

This is only happening on 3.10.5, it wasn't happening on `3.10.0-rc.1` which we were previously testing against. All other tests are passing, so I'm not sure what's causing it?

I'm going to `xfail` that test to get a new release out, but would be great to fix medium term.

**Comment by alexmojaki** [comment]
Released 0.9.1 that should fix this for you.

**Comment by samuelcolvin** [comment]
Thanks so much.
